pkg/sqlite/load.go

```diff
@@ -639,7 +655,21 @@ func (s *SQLLoader) AddBundlePackageChannels(manifest registry.PackageManifest,
 		return fmt.Errorf("csv name not found")
 	}
 
-	if _, err := stmt.Exec(csvName, csvBytes, bundleBytes, bundleImage); err != nil {
+	csv, err := bundle.ClusterServiceVersion()
+	if err != nil {
+		return err
+	}
+
+	skiprange, ok := csv.Annotations[SkipRangeAnnotationKey]
+	if !ok {
+		skiprange = ""
+	}
```

Comment:
What do you think about having a `SkipRange()` method on the `ClusterServiceVersion` struct and handling this there?